robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -198,6 +198,7 @@ public boolean shouldAcquire(String name) {
             || name.startsWith("org.xml.")
             || name.startsWith("org.junit")
             || name.startsWith("org.hamcrest")
+            || name.startsWith("org.scalatest")
```

Comment:
You can achieve the same thing by calling doNotAquireClass("org.scalatest") on the builder... I think I'd prefer not to hard code exclusions for various random libraries in here.